packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -471,6 +478,9 @@ class Uploader {
     const headers = headerSanitize(this.options.headers)
     const reqOptions = { url: this.options.endpoint, headers, encoding: null }
     const httpRequest = request[httpMethod]
+    if (this.options.insecureUpload) {
```

Comment:
what is this option for? 🤔

Reply:
Apologies, I should have diffed this closer, that was a little hack to make this work on our application's dev-server that operates with an expired cert. Not supposed to be included. I'll do another quick look over.

Reply:
@ifedapoolarewaju Pulled out the offending commit.